### PR TITLE
docs(aar): add instructions to use instance profiles with AAR

### PR DESCRIPTION
**Pull request by jat-canonical**
This commit adds the instructions to make use of the newly added support for instance profiles in AAR.

### CODE REVIEW

**Review by morphis — CHANGES_REQUESTED** [review]
LGTM, can you though add a link to the release notes for 1.21 so we can get that linked for people interested?

File: howto/aar/deploy.md [hunk 1]
```diff
@@ -51,17 +53,25 @@ To allow the AAR to access the S3 bucket, create an [IAM](https://docs.aws.amazo
 
 Replace `aar0` in the policy with the name of your bucket.
 
-Once you created the IAM user, create an access key for the user, which the AAR will use. See the [AWS documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for more details on this.
+There are two ways to configure the bucket access for AAR using the policy create above:
```
Comment (keirthana) [inline howto/aar/deploy.md:56]:
```suggestion
There are two ways to configure the bucket access for AAR using the policy create earlier:
```

File: howto/aar/deploy.md [hunk 2]
```diff
@@ -51,17 +53,25 @@ To allow the AAR to access the S3 bucket, create an [IAM](https://docs.aws.amazo
...
 
-Add the credentials to the `config.yaml` file:
+1. Create an IAM user and an access key for this user, which the AAR will use. See the [AWS documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for more details on this. Assign the policy created above to this user.
```
Comment (keirthana) [inline howto/aar/deploy.md:58]:
```suggestion
1. Create an IAM user and an access key for this user, which the AAR will use. See the [AWS documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for more details on this. Assign the policy created earlier to this user.
```

File: howto/aar/deploy.md [hunk 3]
```diff
@@ -51,17 +53,25 @@ To allow the AAR to access the S3 bucket, create an [IAM](https://docs.aws.amazo
...
+2. Create an instance profile using the IAM policy created above and attach the instance profile to the instance where AAR is deployed. See the [AWS documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html) for more details on this.
```
Comment (keirthana) [inline howto/aar/deploy.md:59]:
```suggestion
2. Create an instance profile using the IAM policy created earlier and attach the instance profile to the instance where AAR is deployed. For more information, see the [AWS documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html).
```

**Review by keirthana — CHANGES_REQUESTED** [review]
Some minor suggestions and I see that Simon has already suggested adding the link to RN

**Review by keirthana — APPROVED** [review]
LGTM

**Comment by keirthana** [comment]
@morphis Are you fine with the current changes? Can I go ahead and merge+publish this update?

**Comment by morphis** [comment]
@keirthana if we get a link into 1.21 release notes, then I am fine. However that seems to be still missing from the PR.

**Comment by jat-canonical** [comment]
@morphis They have been added [here](https://github.com/canonical/anbox-cloud-docs/pull/37/files#diff-230f1d2db05c9124e7bbde9fb309b9033633b2c43d87c6e33610ee7b6cb5b4c8R61). Let me know if it needs changes.

**Comment by keirthana** [comment]
@morphis I think there is some confusion here about whether you want the link from the doc to the RN or the reverse. In your most latest comment, you say "into the RN" , so I think it is the way we usually do - from the RN to the docs. Correct?

**Comment by jat-canonical** [comment]
@morphis  could you confirm ?

**Comment by morphis** [comment]
> @morphis I think there is some confusion here about whether you want the link from the doc to the RN or the reverse. In your most latest comment, you say "into the RN" , so I think it is the way we usually do - from the RN to the docs. Correct?

We need to link from the RN to the docs.

**Comment by keirthana** [comment]
Since the pending comment has been addressed, I am merging and publishing this PR.

**Pull request merged**